packages/prisma/migrations/20220715183655_upgrade_to_v4/migration.sql

```diff
@@ -0,0 +1,17 @@
+-- DropForeignKey
+ALTER TABLE "DestinationCalendar" DROP CONSTRAINT "DestinationCalendar_bookingId_fkey";
```

Comment:
I think it's because of this:

<img width="807" alt="image" src="https://user-images.githubusercontent.com/3504472/180019162-57292b37-21ca-40f4-94fc-39183dfb5a02.png">